source/plugin/data/serialization.rst

```diff
@@ -123,3 +125,56 @@ from persistent files using the :doc:`Configurate Library <../configuration/inde
         final DataView container = translator.translate(node);
         return deserializeHealth(container);
     }
+
+
+DataFormat
+==========
+A alternative to DataTranslators is using :javadoc:`DataFormat` which allows you to convert a data container this allows you to store a data container as a hocon, json or even NBT you can of course retrive data using data format.
```

Comment:
This sentence is a bit wonky, and far too long. How about this:
"An alternative to DataTranslators is to use :javadoc:`DataFormat`, which allows you to store a data container as a hocon, json or NBT file. You can also retrieve data using `DataFormat`."
It is also preferable to format class names etc like `this`, wherever they appear.